conans/model/ref.py

```diff
@@ -4,11 +4,23 @@
 from conans.model.version import Version
 
 
+# TODO: Move to some tools module
```

Comment:
Let me know if I leave this code here or move to `conans/utils` module.... or maybe use a six approach: 

```
from six import string_types
isinstance(s, string_types)
```

Reply:
Yes, use six tools, please.